rust/src/lib/query_apply/ethernet.rs

```diff
@@ -140,7 +123,7 @@ impl NetworkState {
                 Some(Interface::Ethernet(cur_iface)),
             ) = (iface, current.interfaces.kernel_ifaces.get(iface.name()))
             {
-                if iface.is_vf_count_changed(cur_iface) {
+                if iface.sriov_is_enabled() || cur_iface.sriov_is_enabled() {
```

Comment:
I carefully reviewed the PR, and it looks good to me, but I am kind of confused by the `iface` and `cur_iface` here, what is the difference between them ? And correct me if I am wrong, does  
`Interfaces.kernel_iface` stand for the interface like loopback interface and  `Interfaces.user_iface` stand for user-defined network interfaces ?

Reply:
The `iface` is desired interface. The `cur_iface` is the current interface.

The `kernel_ifaces` is for interfaces which have kernel representative, like `eth0`, `bond99` and etc.
The `user_ifaces` is  for interfaces which only exist in user space, for example, OVS bridge is user space interface which does not has kernel interface.